snakemake_executor_plugin_slurm/__init__.py

```
@@ -26,18 +27,53 @@
 from snakemake_interface_common.exceptions import WorkflowError
 from snakemake_executor_plugin_slurm_jobstep import get_cpus_per_task
 
-from .utils import delete_slurm_environment
+from .utils import delete_slurm_environment, clean_old_logs
```

Ruff

`.utils.clean_old_logs` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)